### ISSUE TITLE
ReferenceError: JS_ARG is not defined

### ISSUE THREAD

**Issue by hejops** [tridactyl/tridactyl#3763]
The following commands, which I used to use to pass URLs to external scripts (notify-send is just there as an example), no longer work. Did the syntax for this recently change?

```
composite get_current_url | js -p tri.excmds.shellescape(JS_ARG).then(url => tri.excmds.exclaim_quiet('sh /path/to/script ' + url))
composite get_current_url | js -p tri.excmds.shellescape(JS_ARG).then(url => tri.excmds.exclaim_quiet('notify-send ' + url))
```

**Comment by bovine3dom** [comment]
Sorry, we broke it recently without noticing. Thanks for filing the issue! :heart: 

Fixed in f1e7b3cbf81fa0e5d1ab42c09dfe96c6d251d818

(@Rummskartoffel let's agree to stop "fixing" these warnings :rofl: )

**Comment by Rummskartoffel** [comment]
Oh right, those variables are also a thing. I do still think that, in general, we *should* fix these (there's a reason they exist, and I'm pretty sure there's still a way to *actually* do it right), but apparently it'll take a bit more thought.
